src/disease_model/models/auquan_seir.py

```diff
@@ -0,0 +1,300 @@
+"""
+Module for Infections Models
+"""
+
+from scipy import optimize
```

Comment:
Could you have imports keep the package name? It would help readability

Such as
```
import scipy
from scipy import integrate
```